### ISSUE TITLE
TypeScript definitions for GitHub of @actions/github broken

### ISSUE THREAD

**Issue by LukasBombach** [actions/toolkit#365]
**Describe the bug**
The TypeScript definitions for GitHub of `@actions/github` are almost entirely wrong. They export a class with a single public method called `graphql` the takes an object of options.

This does not reflect the documentation or the actual library at all.

The documentation says you can do the following things:

```ts
import { GitHub } from "@actions/github"
const octokit = new github.GitHub(myToken);

octokit.pulls.get(/* ... */);
octokit.issues.create(/* ... */);
octokit.graphql(query, variables);
```

Instead TypeScript says the `octokit` only has a single method called `graphql` expecting a single parameter only.

**To Reproduce**
1. `yarn init -y`
1. `yarn add @actions/github`
1. `touch index.ts`
1. open `index.ts` add the following code
    ```ts
    import { GitHub } from "@actions/github"

    const x: any = null;
    
    const octokit = new github.GitHub(x);

    octokit.pulls.get(x);
    octokit.issues.create(x);
    octokit.graphql(x, x);
    ```

**Expected behavior**
TypeScript will reflect the API of the library and match the documentation

**Desktop (please complete the following information):**
 - OS: macOS Catalina
 - Browser Node.js
 - Version 12 lts

**Comment by LukasBombach** [comment]
The problem appears to be that your generated types import Octokit wrong, like so:

```ts
import Octokit from '@octokit/rest';
```

while it should be imported like this:

```ts
import { Octokit } from '@octokit/rest';
```


not sure why your build does that

**Comment by thboop** [comment]
Hey @LukasBombach , thanks for the report!

What version of @actions/github are you using?
We rolled out a fix for this in `2.1.1`. https://github.com/actions/toolkit/pull/332

I've been unable to reproduce in `2.1.1` with your snippet (with one minor edit)
```
import { GitHub } from "@actions/github"

const x: any = null;

const octokit = new GitHub(x);

octokit.pulls.get(x);
octokit.issues.create(x);
octokit.graphql(x, x);
```

**Comment by LukasBombach** [comment]
Yes! I can confirm it's fixed! Thank you so much! <3
